src/group_theory/submonoid/membership.lean

```diff
@@ -108,7 +108,23 @@ lemma coe_Sup_of_directed_on {S : set (submonoid M)} (Sne : S.nonempty) (hS : di
   (↑(Sup S) : set M) = ⋃ s ∈ S, ↑s :=
 set.ext $ λ x, by simp [mem_Sup_of_directed_on Sne hS]
 
-variables {N : Type*} [monoid N] {P : Type*} [monoid P]
+@[to_additive]
+lemma mem_sup_left {S T : submonoid M} : ∀ {x : M}, x ∈ S → x ∈ S ⊔ T :=
+show S ≤ S ⊔ T, from le_sup_left
+
+@[to_additive]
+lemma mem_sup_right {S T : submonoid M} : ∀ {x : M}, x ∈ S → x ∈ S ⊔ T :=
+show S ≤ S ⊔ T, from le_sup_left
```

Comment:
These lemmas are identical - I assume that's a mistake?

Reply:
Good catch. Fixed